classes/class-gf-securesubmit.php

```diff
@@ -1688,9 +1678,9 @@ private function buildCardHolder($feed, $submission_data, $entry)
      */
     private function buildCheckHolder($feed, $submission_data, $entry)
     {
-        $checkHolder = new HpsCheckHolder();
+        $checkHolder = new ECheck();
         $checkHolder->address = $this->buildAddress($feed, $submission_data, $entry);
```

Comment:
There should not be an address property on `ECheck`, so this line can be removed once the address is added to the authorization builder for check transactions.

Reply:
updated as per review comment